tests/roxctl/bats-tests/local/expect/flavor-interactive.expect.tcl

```diff
@@ -42,6 +42,7 @@ expect "Enter administrator password*:*: " { send "\n" }
 expect "Enter orchestrator (k8s, openshift)*: " { send "k8s\n" }
 expect "Enter default container images settings*:*: " { send "$flavor\n" }
 expect "Enter the directory to output the deployment bundle to*:*: " { send "$out_dir\n" }
+expect "Enter whether to enable telemetry*:" { send "\n" }
```

Comment:
Why would this line move up?

Reply:
The unfortunate answer for now: I don't know yet.
For whatever reason, when adding the declarative config flags, cobra changes the order of flags.

I'll go ahead and merge this for now, but will check whether I can find the reasoning behind the switch-up :/